### PR TITLE
Create test for api post controller

### PR DESCRIPTION
**Pull request by DwinaTech**
Created test for API postController functions and changed deleteOrganisation() function to deleteBranch() also update the delete an organisation route to be ```/api/branch/delete``` instead of ```/api/organisation/delete```

### CODE REVIEW

File: backend/test/post_controller_spec.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          }
+        ]
+      }
+    }
+    const result = await postOrganisation(organisation);
+    expect(result).to.be.a('object')
+    expect(result.org_name).to.equal('Test')
+  })
+
+  it.only('Should edit an organization', async () => {
```
Comment (kkarimi) [inline backend/test/post_controller_spec.js]: remove `.only`

File: backend/test/post_controller_spec.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      ]
+    }
+    const result = await editOrganisation(organisation, orgId, branchId);
+    expect(result).to.be.a('object')
+    expect(result.org_name).to.equal('Test')
+  })
+
+  it('Should delete branch return the number of deleted branch', async () => {
+    const orgId = 1; const branchId = 2;
+    const result = await deleteBranch(orgId, branchId);
```
Comment (kkarimi) [inline backend/test/post_controller_spec.js]: Great!
1) You can add more tests - e.g. there is no test for the route (with request)
2) Usually you would send another request to check if that item exists, and if it gives 404 (so it's deleted and no longer there) then it's another successful test

File: backend/src/routes/api.js [hunk 3]
```diff
@@ -19,10 +19,10 @@ router.post('/', (req, res) => {
   postOrganisation(query).then(responce => res.json(responce));
 });
 
-router.delete('/organisation/delete', async (req, res) => {
+router.delete('/branch/delete', async (req, res) => {
```
Comment (kkarimi) [inline backend/src/routes/api.js]: This shows great and deeper understanding of the project, well done 👍

File: backend/test/post_controller_spec.js (hunk 2 above)
Comment (DwinaTech) [reply backend/test/post_controller_spec.js]: Done

File: backend/test/post_controller_spec.js (hunk 1 above)
Comment (DwinaTech) [reply backend/test/post_controller_spec.js]: removed

File: backend/src/routes/api.js (hunk 3 above)
Comment (DwinaTech) [reply backend/src/routes/api.js]: thank you

File: backend/test/post_controller_spec.js (hunk 2 above)
Comment (ibramoha) [reply backend/test/post_controller_spec.js]: Nice One!!!

**Pull request merged**